lib/clientsidescripts.js

```diff
@@ -107,7 +107,7 @@ functions.findBindings = function(binding, exactMatch, using, rootSelector) {
   function repeaterMatch(ngRepeat, repeater, exact) {
     if (exact) {
       return ngRepeat.split(' track by ')[0].split(' as ')[0].split('|')[0].
```

Comment:
hmm okay, but can we at least add a test for exactRepeater (and/or a comment stating all the forms that the repeater can take) so they know why these splits are necessary?

LGTM otherwise